Documentation/kubelet-wrapper.md

```diff
@@ -61,3 +95,4 @@ ExecStart=/opt/bin/kubelet-wrapper \
 
 [#2141]: https://github.com/coreos/rkt/issues/2141
 [kubelet-wrapper]: https://github.com/coreos/coreos-overlay/blob/master/app-admin/kubelet-wrapper/files/kubelet-wrapper
+[addon-logging]: https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/fluentd-elasticsearch
```

Comment:
This seemed like the best link, other than it being master. I worry that if we point this to a release, we're never gonna remember to update it.